### ISSUE TITLE
[KEYBOARD]: Add to calendar links need an HREF to be keyboard focusable

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#5993]
## Description

The links to add events to a calendar application do not have `HREF` attributes. This means these links cannot receive keyboard focus, and are inaccessible to keyboard and screen reader users. Screenshot attached below.

## Type of Issue
Content. I'm guessing this `.ics` attachment link is being generated by the CMS.
_Per @davidconlon, this may be handled from the front end_

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

**Definition of done**
* [ ] The Add to Calendar links will receive keyboard focus. This can be confirmed by pressing `TAB` and verifying the link has the yellow focus halo.
* [ ] Add to Calendar link can be verified by opening the dev tools console and typing the following at the `>` prompt:
```javascript
document.activeElement
```

## Environment

* MacOS Mojave
* Chrome latest
* https://www.va.gov/pittsburgh-health-care/events/veterans-town-hall-0/ <br/>There will be others, anywhere the `.ics` attachment is being generated.

## Possible Fixes (optional)
Add an HREF attribute to the links:

```html
<a
  data-description="The VA Pittsburgh Healthcare System Director invites you to attend our Veterans Town Hall."
  data-end="2019-09-23T15:30:00"
  data-location="University Drive C Pittsburgh, PA"
  data-start="2019-09-23T14:00:00"
  data-subject="Veterans Town Hall"
  href="https://some-unique-url-to-ics-attachment"
  id="add-to-calendar-link"
>
  <i class="va-c-social-icon fas fa-calendar-check vads-u-margin-right--0p5"></i>
  Add to Calendar
</a>
```

## WCAG or Vendor Guidance (optional)

* [Keyboard: Understanding SC 2.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html)

## Screenshots or Trace Logs

![Screen Shot 2020-02-19 at 11 34 56 AM](https://user-images.githubusercontent.com/934879/74859907-6dcc3080-530d-11ea-92c3-25349d2341ac.png)

**Comment by kelsonic** [comment]
![image](https://user-images.githubusercontent.com/12773166/78830844-de0a4080-79a5-11ea-91b5-48685c22c41f.png)

It looks like this can be done by editing this file in `vets-website`: [`src/site/includes/social-share.drupal.liquid`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/includes/social-share.drupal.liquid)

I'd be happy to pick this up! @MickinSahni what do you think about adding this to our plate over on public-websites?

**Comment by MickinSahni** [comment]
> It looks like this can be done by editing this file in `vets-website`: [`src/site/includes/social-share.drupal.liquid`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/includes/social-share.drupal.liquid)
> I'd be happy to pick this up! @MickinSahni what do you think about adding this to our plate over on public-websites?

Hey @kelsonic, sure we could do that, but looks like a template issue. Any issues with PW getting into that?

**Comment by kelsonic** [comment]
> Any issues with PW getting into that?

@MickinSahni I don't think so! Let's do it! 🔥

**Comment by ethanteague** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/12378

**Comment by 1Copenut** [comment]
@mmiddaugh && @ethanteague && @jenstrickland I've verified this one passes in Staging. Please feel free to close this issue ticket at your convenience.

**Comment by mmiddaugh** [comment]
@jenstrickland  this one is ready for you

**Comment by jenstrickland** [comment]
Validated. 

* [X] The Add to Calendar links will receive keyboard focus. This can be confirmed by pressing `TAB` and verifying the link has the yellow focus halo.
* [X] Add to Calendar link can be verified by opening the dev tools console and typing the following at the `>` prompt:
```javascript
document.activeElement
```
